.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
Do we want to check in the editor config?!

Reply:
https://editorconfig.org/ is a standard that is supported by most editors. You can see it as a way to have default config for editors, e.g. space/tabs, numbers of spaces, .. It's kind of a companion to code linters. 
I feel like it's common to check this in those days?

Reply:
I see. Makes sense then. I'm also thinking to config ESlint later as we move forward.